### PR TITLE
Test base fixes

### PR DESCRIPTION
**Pull request by jim-minter**

### CODE REVIEW

**Comment by jim-minter** [comment]
@mjudeikis ptal

**Comment by jim-minter** [comment]
test-base buildconfig changes are already live

File: ci-operator/config/openshift/openshift-azure/openshift-openshift-azure-master.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 base_images:
   base:
     cluster: https://api.ci.openshift.org
-    name: plugin-base
-    namespace: azure
-    tag: latest
+    name: rhel
```
Comment (mjudeikis) [inline ci-operator/config/openshift/openshift-azure/openshift-openshift-azure-master.yaml]: This is new. we have rhel 7 now here!

File: projects/azure/base-images/test-base.yaml [hunk 2]
```diff
@@ -28,72 +28,63 @@ items:
...
+        RUN go get golang.org/x/tools/cmd/goyacc
+        RUN go get github.com/golang/mock/mockgen
+        RUN go get github.com/Masterminds/glide
+
+        # pin go-bindata to 41975c0, master got broken on 1/4/2019
+        RUN go get -d github.com/go-bindata/go-bindata/go-bindata
+        RUN cd /go/src/github.com/go-bindata/go-bindata && git checkout -q 41975c0
+        RUN go install github.com/go-bindata/go-bindata/go-bindata
+
+        # pin goimports to 04b5d21e, see discussion https://github.com/openshift/openshift-azure/pull/862
```
Comment (mjudeikis) [inline projects/azure/base-images/test-base.yaml]: I tempted to start migrating our code to use the new goimport format. Considering half of our code is already done

**Pull request merged**